lib/ctbl.gi

```diff
@@ -4202,6 +4203,52 @@ InstallMethod( BrauerTableOp,
         # (This function takes care of a class permutation in `tbl'.)
         result:= CharacterTableIsoclinic( modtbls, tbl );
       fi;
+    else
+      ppart:= 1;
+      n:= Size( tbl );
+      while n mod p = 0 do
+        ppart:= ppart * p;
+        n:= n / p;
+      od;
+      if ppart in OrdersClassRepresentatives( tbl ) then
+        # The Sylow 'p'-subgroup is cyclic.
```

Comment:
I suggest to remove quotes around `p` to ease reading.

Reply:
@ThomasBreuer just to explain why: I read that first as `p'-subgroup`, then reread to figure out what does that mean. The style you use here is not consistent with other styles used in the GAP library: for example we have

```
grp.gi:      # compute generators for the torsion Omega p-subgroups of the center
grpprmcs.gi:##  O_p(G), the p-core of G, is the maximal normal p-subgroup
grpprmcs.gi:                        # abelian normal p-subgroup of G
grpprmcs.gi:    # p-subgroup of G; taking commutator subgroups, find abelian normal
grpprmcs.gi:    # p-subgroup of G.
grpprmcs.gi:    # p-subgroup; kernel is abelian p-group. Take image at this action, and
```
and also one case of
```
ctblsolv.gi:    # Choose a normal elementary abelian `p'-subgroup `N',
```